samples/zigbee/network_coordinator/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+  set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+endif()
```

Comment:
Remove those lines and rename `dts.overlay` to `app.overlay`

Reply:
done